.github/workflows/ci.yml

```diff
@@ -31,3 +31,12 @@ jobs:
             # NOTE: Lint needs to run after build because it checks the compiled files for types
             - name: Lint
               run: pnpm lint
+
+            - name: Typecheck idkit-core
+              run: cd packages/core && pnpm tsc
+
+            - name: Typecheck idkit
+              run: cd packages/react && pnpm tsc
+
+            - name: Typecheck idkit-standalone
+              run: cd packages/standalone && pnpm tsc
```

Comment:
Doesn't `tsc` get automatically run on build?

Reply:
for some reason some tsc issues were not being picked up in examples when building